org.civicrm.bootstrapcivicrm/scss/civicrm/_mixins.scss

```diff
@@ -15,4 +15,18 @@
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box;
+}
+
+// Generic Transform
```

Comment:
Please standarize annotation description for all mixins also please try to give more description instead of just saying 'Translate'